cpp/include/cudf/stream_compaction.hpp

```diff
@@ -283,11 +278,11 @@ std::unique_ptr<table> unordered_drop_duplicates(
  * @param[in] null_handling flag to include or ignore `null` while counting
```

Comment:
Docstring above this needs fixed to avoid using "distinct" in the `unique_count` definition:
```diff
- * @param[in] input The column_view whose number of distinct consecutive groups will be counted
+ * @param[in] input The column_view whose consecutive groups of equivalent rows will be counted
```